### PR TITLE
Gets rids of spawns in airlock code

### PR DESCRIPTION
**Pull request by ghost**
I AM TRULY LUSTY FOR GBP

### CODE REVIEW

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -792,6 +795,13 @@
 		..()
 	return
 
+/obj/machinery/door/airlock/proc/electrified_loop()
+	while (secondsElectrified > 0)
```
Comment (kevinz000) [inline code/game/machinery/doors/airlock.dm]: This process seems wrong .

Comment (MoreRobustThanYou) [reply code/game/machinery/doors/airlock.dm]: baaaa

**Comment by Anonmare** [comment]
Such a lust for points... 
WHO?

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -325,33 +327,34 @@
 	if(src.secondsMainPowerLost > 0)
 		src.secondsMainPowerLost = 0
 
+/obj/machinery/door/airlock/proc/handlePowerRestore()
+	var/cont = 1
```
Comment (ChangelingRain) [inline code/game/machinery/doors/airlock.dm]: TRUE/FALSE

**Review by Cyberboss — APPROVED** [review]
As much as I hate these shitty loops, I hate spawn more

**Pull request merged**